src/sider/sykmeldinger/sykmelding/sykmeldingOpplysninger/SykmeldingSeksjon.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { ReactNode } from "react";
+import { Heading } from "@navikt/ds-react";
+
+interface SykmeldingSeksjonProps {
+  tittel: string;
+  children?: ReactNode;
+}
+
+export function SykmeldingSeksjon(
+  sykmeldingCheckboxProps: SykmeldingSeksjonProps
```

Comment:
Man kan pakke props ut i input parameteret her. Så slipper du linjen under her 😊
```suggestion
  { tittel, children }: SykmeldingSeksjonProps
```

Reply:
Likte det bedre, så skal endre.